indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -51,8 +58,96 @@ class RHAbstractsDownloadAttachment(RHAbstractsReviewBase):
     }
 
     def _checkParams(self, params):
-        RHAbstractsReviewBase._checkParams(self, params)
+        RHAbstractReviewBase._checkParams(self, params)
         self.abstract_file = AbstractFile.get_one(request.view_args['file_id'])
 
     def _process(self):
         return self.abstract_file.send()
+
+
+class RHListOtherAbstracts(RHConferenceBaseDisplay):
```

Comment:
It's not very obvious what this RH is supposed to be used for. Maybe a docstring wouldn't harm.